src/components/search.jsx

```diff
@@ -65,26 +62,15 @@ function sendSearch(query, apiCategories) {
     index.search(query, options)
       .then(({ hits }) => {
         const entries = hits.map((hit) => hitToAPI(hit))
-          .filter(([, entry]) => !entry.regions || entry.regions.includes(region));
-        const categories = {};
-
-        entries.forEach((entry) => {
-          for (const category of entry[1].categories) {
-            if (!(category in categories)) categories[category] = [];
-            categories[category].push(entry);
-          }
-        });
+          .filter(([, entry]) => !entry.regions || entry.regions.includes(region))
+          .sort(([a,], [b,]) => a.localeCompare(b));
 
         if (entries.length !== 0) {
-          const tables = <>
-            {Object.keys(categories).sort().map((category, index) =>
-              <Table Category={category} Title={apiCategories[category].title} Order={index} search={categories[category]} />
-            )}
-          </>;
+          const table = <Table Category="search" Title="Search Results" search={entries} grid="" />
```

Comment:
Should the `grid` parameter have a value? It seems to work fine without it.

Reply:
That shouldn't be necessary if no category buttons are present.